sdk/python3/ccnp/measurement/measurement_sdk.py

```diff
@@ -63,6 +74,7 @@ def get_measurement(self):
             return ""
 
         LOG.info("Fetch measurement successfully.")
+        self.cleanup_channel()
```

Comment:
Is it safe to clean up the channel here? Such as get_measurement more than once for one instance.

Reply:
> Is it safe to clean up the channel here? Such as get_measurement more than once for one instance.
I provided the new initialization of the class each time user calls get_platform_measurement(). So the clean up works fine here. But is it better to make this get_measurement() with the staticmethod decorator to avoid user calling from the instance of the class?